validation/coverage/coverage.1.extra_field.fail.json

```diff
@@ -0,0 +1,8 @@
+{
+  "appVersion": "63.0a1",
+  "appUpdateChannel": "nightly",
+  "osName": "Darwin",
+  "osVersion": "17.7.0",
+  "telemetryEnabled": true,
+  "dummy": true
+}
```

Comment:
It seems some validation payloads in the repo have a newline and some don't. For consistency we should either have newlines or not, but for the purposes of this PR it is irrelevant.

Reply:
I filed #200 to follow up on this.